src/wmnds/assets/sass/_utilities.scss

```diff
@@ -21,6 +21,26 @@
   }
 }
 
+//Group buttons 
+.wmnds-group-buttons {
+
+  & > *:not(:first-child) {
+    margin-top: $size-md;
+  }
+
+  @media screen and (min-width: $breakpoint-md) {
+    &.wmnds-grid {
+      flex-flow: row nowrap;
+    }
+    
+    & > *:not(:first-child) {
+      margin-top: 0;
+      margin-left: $size-md;
+    }
+  }
+
+}
+
```

Comment:
I tried to do something generic as I don't have any cookies.scss file and as I think this could be reused.
This will group 2 or more buttons with a space in between (this way you can have 2 col-6 with a space between, 2 buttons with same width and in the same line, same for 3, 4, 5 buttons).
On mobile it wraps, bigger screens doesn't wrap. 

I felt it could be reused but if you maintain your opinion I can move it and rename it. 👍